File: src/main/java/ru/javarush/cryptoanalyzer/konovalov/data/Alphabet.java
package ru.javarush.cryptoanalyzer.konovalov.data;

import java.util.List;

public class Alphabet {
    private Alphabet(){

    }

    static String ENGLISH_LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    static String ALL_ALPHABET = ENGLISH_LETTERS.toLowerCase();

    static char[] CRYPT_ALPHABET_ARRAY = ALL_ALPHABET.toCharArray();

    static List<Character> getAlphabetList() {
        return alphabetList;
    }

    static List<Character> alphabetList = ENGLISH_LETTERS.toLowerCase().chars().mapToObj(c -> (char) c).toList();

    public static char[] getAlphabet() {
        return CRYPT_ALPHABET_ARRAY;
    }

    public static int getAlphabetLength() {
        return CRYPT_ALPHABET_ARRAY.length;
    }


}

File: src/main/kotlin/data/Alphabet.kt
package data

import java.util.*
import java.util.stream.Collectors


var ENGLISH_LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
var ALL_ALPHABET = ENGLISH_LETTERS.lowercase(Locale.getDefault())
var CRYPT_ALPHABET_ARRAY = ALL_ALPHABET.toCharArray()
var alphabetList = ENGLISH_LETTERS
    .lowercase(Locale.getDefault())
    .chars()
    .mapToObj(Int::toChar)
    .collect(Collectors.toList())

fun getAlphabetList(): List<Char> {
    return alphabetList
}
fun getAlphabet(): CharArray {
    return CRYPT_ALPHABET_ARRAY
}
fun getAlphabetLength(): Int {
    return CRYPT_ALPHABET_ARRAY.size
}